build.gradle

```diff
@@ -58,6 +58,7 @@ plugins {
     id 'eclipse'
     id 'scala'
     id 'maven-publish'
+    id 'base'
```

Comment:
This shouldn't be needed, the base plugin is a dependency of at least java-library